### DISCUSSION TITLE
Problem scaling with pblum_mode=absolute

### DISCUSSION THREAD

**Discussion by rfreire-400** [phoebe-project/phoebe2#872] [Questions]
I'm preparing my data to sample using EMCEE and Dynesty. Since it's best to sample with plum_mode = absolute, I'm trying to set that up but I'm having some scaling issues. Notably, there's a 10^7 difference between the synthetic and real light curves. I've looked into the documentation and it recommends modifying limb darkening, irradiation, distance and third light, which can all affect fluxes. While there are differences when I apply those parameters, it does not account for a difference of 10^7. 

Is there a parameter here that I'm forgetting to set up? I've tested this with two systems, both of which ended up with a difference in the 10^6 - 10^7 range. Because of that, I'm tempted to believe that there's a parameter I'm missing.

Any advice would be greatly appreciated. Thanks!
<img width="905" alt="Screenshot 2024-05-21 at 10 22 48 AM" src="https://github.com/phoebe-project/phoebe2/assets/113457579/31d03452-1d86-4645-967a-975cc9ce26a5">

<img width="905" alt="Screenshot 2024-05-21 at 10 25 29 AM" src="https://github.com/phoebe-project/phoebe2/assets/113457579/726901fb-ddc9-4f7e-a5d4-2b3c50acd0dd">

**Comment by kecnry** [comment]
Are your observational data calibrated and actually in `W/m**2`?  It looks like you have some code to convert between `e/s` to `W/m**2` but then don't actually convert the input arrays?

**Reply by rfreire-400** [reply]
Yes, I have some code that converts e/s to W/m^2 on a different notebook. I save the data as a .csv file so I don't have to calculate it every time I want to run this.

I was testing this with e/s and changing the inter_weighting parameter to see if that would help at all. I forgot to erase some of that old code.

**Reply by kecnry** [reply]
By using your inputs with the Stefan-Boltzmann law and assuming spherical stars:

```
import numpy as np
from astropy import units as u
from astropy import constants as c

L1 = 4 * np.pi * (7.785*u.solRad)**2 * c.sigma_sb * (9019*u.K)**4
L2 = 4 * np.pi * (2.14*u.solRad)**2 * c.sigma_sb * (5329*u.K)**4
d = (570*u.pc).to(u.solRad)

flux = (L1+L2)/(4*np.pi)/(d**2)
```

gives `<Quantity 3.59035398e-11 W / m2>`.  This is obviously making a lot of assumptions, but seems consistent with the results from phoebe given the inputs.  Is it possible that one of those inputs (temperature, radius, distance - as those will most strongly affect the scaling of the fluxes) is not correct or that the data or unit conversion of the data in incorrect?

**Reply by rfreire-400** [reply]
For this first system, yes it is certainly possible that the values for temperature and radius are incorrect. Those values were found using estimators and optimizers when pblum_mode was set to absolute. 
I do have a second system with a lot of existing literature. Even with the values from that system, the difference is the same (linked below). 

I'll double check the unit conversion and get back to you.

<img width="879" alt="Screenshot 2024-05-21 at 1 07 45 PM" src="https://github.com/phoebe-project/phoebe2/assets/113457579/2c79530a-8b91-42ea-be9c-951737cd02d0">

<img width="879" alt="Screenshot 2024-05-21 at 1 08 13 PM" src="https://github.com/phoebe-project/phoebe2/assets/113457579/4a901d61-0f03-401e-802c-c5e1a7d083b5">

**Reply by rfreire-400** [reply]
I've checked the conversion factor and there was indeed an error with it, thank you for the advice. I got the synthetic values for both systems much closer to the observed values. There's still a gap, but I think that it can be fixed. So I should focus on changing the temperature, radius and distance to see the greatest difference?


<img width="879" alt="Screenshot 2024-05-21 at 3 18 24 PM" src="https://github.com/phoebe-project/phoebe2/assets/113457579/5d8a2c36-aef3-4793-ad47-6f60862bce23">
<img width="879" alt="Screenshot 2024-05-21 at 3 18 53 PM" src="https://github.com/phoebe-project/phoebe2/assets/113457579/f68899d5-a025-463d-a3ea-e2bdfd602bff">

**Reply by kecnry** [reply]
Those are going to be very degenerate without some additional information/priors, but yes, flux will approximately scale with `R**2 T**4 / d**2`

**Reply by rfreire-400** [reply]
Sounds good, thank you for your help
